src/ngx_http_lua_control.c

```diff
@@ -319,11 +319,18 @@ ngx_http_lua_ngx_exit(lua_State *L)
                                | NGX_HTTP_LUA_CONTEXT_CONTENT
                                | NGX_HTTP_LUA_CONTEXT_TIMER
                                | NGX_HTTP_LUA_CONTEXT_HEADER_FILTER
+                               | NGX_HTTP_LUA_CONTEXT_BODY_FILTER
                                | NGX_HTTP_LUA_CONTEXT_BALANCER
                                | NGX_HTTP_LUA_CONTEXT_SSL_CERT
                                | NGX_HTTP_LUA_CONTEXT_SSL_SESS_STORE
                                | NGX_HTTP_LUA_CONTEXT_SSL_SESS_FETCH);
 
+    /* rc is useless in bodyfilter phase */
```

Comment:
@agentzh Can we do this: just log an error log ([like this one](https://github.com/openresty/lua-nginx-module/blob/master/src/ngx_http_lua_control.c#L371)) in [here](https://github.com/rainingmaster/lua-nginx-module/blob/bodyfilter_exit/src/ngx_http_lua_control.c#L333). Because the difference between the output of NGX_OK or NGX_ERROR in `ngx_http_lua_body_filter_by_chunk` is only the output of the most have a buf, the user may be difficult to see the difference? 
Now, once the user calls the `ngx.exit()` in body fitler, it will output the last buf and exit.